integration-tests/tests/integration/main.rs

```diff
@@ -939,10 +1074,20 @@ fn create_archive(
 }
 
 fn run_archive(archive_file: &Utf8Path) -> (TempProject, Utf8PathBuf) {
+    run_archive_with_args(archive_file, RunProperty::Relocated as u64, None)
+}
+
+fn run_archive_with_args(
+    archive_file: &Utf8Path,
+    run_property: u64,
+    expected_exit_code: Option<i32>,
```

Comment:
These tests seem great.

Why use `Option<i32>` and not explicitly pass in `NextestExitCode::TEST_RUN_FAILED`?

Reply:
Out of habit I use `Option`'s in test helpers. I removed the option and now pass in the i32 directly.